.github/workflows/main.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
+        node-version: [10.x, 12.x, 14.x, 16.x]
```

Comment:
I have no doubt there is a reason, but out of interest, why are you building against all these versions instead of just 16?

Reply:
It's because our main distribution is the raw sass files so we test that they compile on all LTS versions. Otherwise we could push an update that isn't compatible with node-sass on 12.x and break a downstream app

I'm not sure that's really a possibility, but it feels like a reasonable bit of due diligence